lms/static/sass/course.scss.mako

```diff
@@ -41,10 +41,7 @@
 @import 'course/courseware/sidebar';
 @import 'course/courseware/amplifier';
 
-## Import styles for search
-% if env["FEATURES"].get("ENABLE_COURSEWARE_SEARCH"):
-    @import 'search/_search';
-% endif
+@import 'search/_search';
```

Comment:
@marcotuts yes. courseware search is disabled currently. I enabled the sass because of sharing.